### ISSUE TITLE
Ztunnel doesn't work well with out VM simulation tests

### ISSUE THREAD

**Issue by howardjohn** [istio/istio#54338]
We run VMs by having a pod and a WE pointing to it. The pod simulates a VM.

The problem is this ends up with 2 workloads for 1 IP, which ztunnel gets confused by. This ultimately means we cannot test Ztunnel --> Fake VM

**Comment by keithmattix** [comment]
Why doesn't un-enrolling the pod fix this? Or can a workload entry not point to an un-enrolled pod?

**Comment by howardjohn** [comment]
Its not related to whether its in the mesh, the problem is we get 2 WDS workloads for the same IP I think. Actually TBH I am not 100% sure but ultimately we end up with 0 endpoints for the service in ztunnel

**Comment by keithmattix** [comment]
Ah, so WDS doesn't filter based on enrollment? Interesting...it's almost like we need to exclude the pod from service discovery or merge WE and Pod together, but that sounds really hacky

**Comment by bleggett** [comment]
> Ah, so WDS doesn't filter based on enrollment? 

Yeah - my understanding would be that WDS would not have a workload entry for a pod that is entirely excluded from Istio? I'm not sure why we would have 2 entries, or why this would break.
